IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10: Error while making the assignments

### Description of the issue

*IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10* had an issue with assignments and this was "fixed" by #3947, nevertheless it introduced a bug by mistyping cmdlet *New-MgBetaDeviceManagementDeviceConfigurationAssignmentAssignment* here https://github.com/microsoft/Microsoft365DSC/blob/0d2935f2ed5cf7d7e2f0ed26e8259389e4106d9b/Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10/MSFT_IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10.psm1#L687C1-L688C1.

Fix is just to remove the extra Assignment word in the cmdlet.

### Microsoft 365 DSC Version

1.23.1129.1

### Which workloads are affected

other

### The DSC configuration

```powershell
N/A
```


### Verbose logs showing the problem

```powershell
2023-12-02T15:12:21.5791537Z ##[error]The term 'New-MgBetaDeviceManagementDeviceConfigurationAssignmentAssignment' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```


### Environment Information + PowerShell Version

```powershell
N/A
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,9 @@
 
 # UNRELEASED
 
+* IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10
+  * Fix typo in assignment cmdlet
+    FIXES [#3996](https://github.com/microsoft/Microsoft365DSC/issues/3996)
 * DEPENDENCIES
   * Updated MSCloudLoginAssistant to version 1.1.1.
 

--- a/Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10/MSFT_IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10.psm1
+++ b/Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10/MSFT_IntuneWindowsUpdateForBusinessRingUpdateProfileWindows10.psm1
@@ -684,7 +684,7 @@ function Set-TargetResource
         }
         foreach ($assignment in $intuneAssignments)
         {
-            New-MgBetaDeviceManagementDeviceConfigurationAssignmentAssignment `
+            New-MgBetaDeviceManagementDeviceConfigurationAssignment `
                 -DeviceConfigurationAssignmentId $policy.id `
                 -BodyParameter $assignment
         }

